README.md

```diff
@@ -5,3 +5,10 @@ Wiki is [available online](https://github.com/OpenSC/OpenSC/wiki)
 Please take a look at the documentation before trying to use OpenSC.
 
 [![Travis CI Build Status](https://travis-ci.org/OpenSC/OpenSC.svg)](https://travis-ci.org/OpenSC/OpenSC/branches) [![AppVeyor CI Build Status](https://ci.appveyor.com/api/projects/status/github/OpenSC/OpenSC?branch=master&svg=true)](https://ci.appveyor.com/project/LudovicRousseau/OpenSC/branch/master) [![Coverity Scan Status](https://scan.coverity.com/projects/4026/badge.svg)](https://scan.coverity.com/projects/4026)
+
+Build and test status of specific cards:
+
+| Cards                 | Status |
+|-----------------------|--------|
+| CAC                   | [![CAC](https://gitlab.com/redhat-crypto/OpenSC/badges/cac/build.svg)](https://gitlab.com/redhat-crypto/OpenSC/pipelines) |
+| Coolkey               | [![Coolkey](https://gitlab.com/redhat-crypto/OpenSC/badges/coolkey/build.svg)](https://gitlab.com/redhat-crypto/OpenSC/pipelines) |
```

Comment:
These URLs show a "500 Server Error"

Reply:
It was probably momentary error when you tried. Now they seem to work fine.